js/renderer.js

```diff
@@ -29,29 +29,35 @@ class Renderer {
 
         for (layer of bg.layers) {
             const img = document.getElementById(layer.image);
-            this.fillCanvasWithImage(img, layer.x, layer.y);
+            const zoomFactor = (this.camera.zoom) / (Math.pow(this.camera.zoom, 1 - layer.relativeSpeed));
+            // pretty sure this is wrong somehow
+            const x = Math.round((layer.x * zoomFactor) + (this.camera.zoomOffsetX * layer.relativeSpeed));
```

Comment:
i think i might need to remove math.round here. its causing some weird jitter during zoom, and idk if its even boosting performance since the images are scaled weirdly

Reply:
I barely notice the jitter on my end. Is removing the rounding affecting the rendering time in any significant manner? I know it would be doing subpixel rendering, but I would think it'd be hard to smooth out the jitter without that.

Reply:
the thing is i think its already having to do subpixel rendering because its not the native width/height during zoom, but im not positive